package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "template-uuid-instantiator",
+  "version": "1.0.0",
+  "description": "",
+  "main": "main.js",
+  "type": "module",
+  "scripts": {
+    "test": "jest",
+    "build": "tsc && vite build",
+    "lint": "eslint src"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Looks like the default license is MIT. We should include the license file attributed (I guess) to Vlaamse Overheid (at least this is how the other repos are).